LICENSE

```diff
@@ -12,8 +12,8 @@ Licensor:             Obol Labs, Inc.
 Licensed Work:        Charon, the Distributed Validator Client
                       The Licensed Work is © 2022-2023 Obol Labs, Inc.
 
-Additional Use Grant: The use of the Licensed Work by the Licensee is limited
-                      to one distributed validator on a production network.
+Additional Use Grant: Any uses listed and defined at
+                      charon-license-grants.obol.eth
```

Comment:
This ENS probably doesn't have any details associated with it?..
![Uploading Screenshot 2023-04-10 at 16.47.57.png…]()

Reply:
Yeah I need Collin to give me admin access atm